hazelcast/src/test/java/com/hazelcast/logging/Log4jLoggerTest.java

```diff
@@ -41,7 +41,7 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.verifyNoInteractions;
 
-@RunWith(HazelcastParallelClassRunner.class)
+@RunWith(HazelcastSerialClassRunner.class)
```

Comment:
What do you think about keeping it parallel and instead replacing `LOG_EVENT` and `LOG_EVENT_OFF` fields with `AbstractLoggerTest#someEvent` method:
```java
static LogEvent someEvent(Level level, String loggerName) {
        LogRecord logRecord = new LogRecord(level, MESSAGE);
        logRecord.setThrown(THROWABLE);
        logRecord.setLoggerName(loggerName);
        return new LogEvent(logRecord, new SimpleMemberImpl());
}
```
and then use it like this:
```diff
    @Test
    public void logEvent_shouldLog() {
-       LogRecord logRecord = LOG_EVENT.getLogRecord();    
-       Log4jTrackingAppender trackingAppender = registerTrackingAppender(logRecord.getLoggerName());
+       String loggerName = "logEvent_shouldLog";
+       LogEvent logEventWarn = someEvent(Level.WARNING, loggerName);
+       Log4jTrackingAppender trackingAppender = registerTrackingAppender(loggerName);
        trackingAppender.assertNoLoggedEvents();

        hazelcastLogger.log(logEventWarn);

        verifyNoInteractions(mockLogger);
        trackingAppender.assertNumberOfLoggedEvents(1);
    }
```